### ISSUE TITLE
New Windows build system

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#1889]
DTU is giong to nuke the build server that we use for CoolProp.

I can set up  a new machine, but isn't this a good opportunity to migrate to AppVeyor? I do not think that we use proprietary software for the Windows builds anymore.

**Comment by ibell** [comment]
Yes, I agree that migration to AppVeyor/Travis would be great and I could recover my Mac mini to do something more useful.  How about a version 6.4 release after I get my build slave fired up again and then after that: **drop Python 2.7 support**.  That would free us up to use C++11 (finally)

**Comment by jowr** [comment]
Yes, good idea. Fixing the exception-based flow control and moving to C++11 could be great. Let us find out what is achievable regarding the very limited amount of time that I can use on CoolProp these days.

**Comment by jowr** [comment]
I have started playing with GitHub actions - maybe we can retire our builders... https://github.com/CoolProp/CoolProp/actions

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
